machines/README.md

```diff
@@ -0,0 +1,26 @@
+# Vagrantfiles for WSO2 Identity Server
+
+This section defines the procedure to execute run Vagrant resources for a setup of WSO2 Identity Server single
+node with Analytics support.<br>
+
+Please note that in order to run these Vagrant resources use, you need to install
+[Oracle VM VirtualBox](http://www.oracle.com/technetwork/server-storage/virtualbox/downloads/index.html),
+as Vagrant uses Oracle VM VirtualBox as the default provider.
+
+## How to run the Vagrantfiles
+
+##### 1. Checkout this repository into your local machine using the following Git command.
+```
+git clone https://github.com/wso2-incubator/vagrant-is.git
+```
+
+##### 2. Build and add the Vagrant boxes for external MySQL database, WSO2 Identity Server and WSO2 Identity Server Analytics using the box automation resources provided, [here](../../boxes).
```

Comment:
Where do we have instructions for creating the vagrant boxes?

Reply:
@imesh the contents within the path directed by this link is not available within this pull request. I will remove this line.